pkg/utils/panic_handler.go

```diff
@@ -33,21 +35,33 @@ func HandlePanics(ctx context.Context, result *reconcile.Result) {
 
 			if err.Error() == "runtime error: invalid memory address or nil pointer dereference" {
 				logger.Error(err, "Recovered from a nil pointer dereference or invalid memory address error")
```

Comment:
Would it be possible to add this error message (or even better, the original panic error) to the CR? I think it would be helpful to not only see 'pod X crashed 5 minutes ago', but also get some information about why it crashed. Of course, it can be read from the logs, but if I check the CR anyway, it would be easier to get the information from there.

Reply:
This is a good idea. I have added a description containing the error text. If required, we can later further extend the entries.